.nsprc

```diff
@@ -1,6 +1,6 @@
 {
   "exceptions": [
-    "https://nodesecurity.io/advisories/577",
-    "https://nodesecurity.io/advisories/157"
+    "https://npmjs.com/advisories/577",
+    "https://npmjs.com/advisories/157"
   ]
-}
\ No newline at end of file
+}
```

Comment:
Ah last thing - would it make sense to rename this file to something else now?

Reply:
Yeah, probably. Does it even need to be its own file?

Reply:
Probably not, unless we're going to add security exceptions all the time 😆